### PR TITLE
Company logo url changed for DhiWise in Jobs

### PR DESCRIPTION
**Pull request by rahul-shingala**

### CODE REVIEW

File: public/data/job-openings-data.json [hunk 1]
```diff
@@ -448,7 +448,7 @@
       "companyName": "DhiWise Pvt. Ltd.",
       "location": "Surat, India",
       "bio": "DhiWise is a programming platform where developers can convert their designs into developer-friendly code for mobile and web apps. DhiWise automates the application development lifecycle and instantly generates readable, modular and reusable code. At DhiWise, you can fall in love with your work again. A platform built for developers to build apps in a way that uses their engineering muscle rather than forcing the use of crammed information.",
-      "companyLogoUrl": "https://drive.google.com/file/d/1CZ2lSzbhLWTIeG6fsZ5UQYl8_qzCshdK/view",
+      "companyLogoUrl": "https://uploads-ssl.webflow.com/618e36726d3c0f19c9284e56/6392ef0773801c4452bd8bde_DhiWise-Logo-150X150.jpgw",
```
Comment (JaldeepAsodariya) [inline public/data/job-openings-data.json:451]: Access denied, Please check at your end and then submit in PR.
Thanks

**Review by JaldeepAsodariya — CHANGES_REQUESTED** [review]

File: public/data/job-openings-data.json (hunk 1 above)
Comment (JaldeepAsodariya) [reply public/data/job-openings-data.json:451]: Screenshot 2022-12-09 at 4 32 35 PM

Comment (PareshMayani) [reply public/data/job-openings-data.json:451]: @JaldeepAsodariya I think the issue is typo in URL, "W" got appended at last https://uploads-ssl.webflow.com/618e36726d3c0f19c9284e56/6392ef0773801c4452bd8bde_DhiWise-Logo-150X150.jpg

Comment (rahul-shingala) [reply public/data/job-openings-data.json]: [typo error removed]

**Review by JaldeepAsodariya — APPROVED** [review]

**Pull request merged**